frontend/ui/data/strings.lua

```diff
@@ -77,8 +77,10 @@ S.HIGH = _("high")
 S.ZERO_DEG = _("0 deg")
 S.FIVE_DEG = _("5 deg")
 S.TEN_DEG = _("10 deg")
-S.PORTRAIT = _("portrait")
-S.LANDSCAPE = _("landscape")
+S.PORTRAIT = _("up")
```

Comment:
I'm not quite sure what I think of up/down/left/right, but either way such generic strings should probably be context-disambiguated:
```suggestion
S.PORTRAIT = C_("Rotation", "up")
```
http://koreader.rocks/doc/modules/gettext.html#pgettext

Reply:
didnt know about this, thanks

Reply:
The LinuxFB convention is Upright, Clockwise, Upside Down, Counter-Clockwise. I'm kinda use to it by now, but it's potentially not really any better, so, just throwing it out there ;).

Reply:
> The LinuxFB convention is Upright, Clockwise, Upside Down, Counter-Clockwise. I'm kinda used to it by now, but it's potentially not really any better, so, just throwing it out there ;).

those strings are long and won't fit well in the UI toggles. thats why I made it short and simple